contracts/src/examples/PingPong.sol

```diff
@@ -47,6 +49,25 @@ contract PingPong is XApp {
         _xpingpong(xmsg.sourceChainId, xmsg.sender, times, n - 1);
     }
 
+    /**
+     * @notice The pingpong xmsg loop
+     * @dev Used to test differnce in gas usage between xrecv and non-xrecv functions
+     * @param times The pingpongs in the loop
+     * @param n The number of xcalls left to make
+     */
+    function pingpong_norecv(uint64 times, uint64 n) external {
```

Comment:
Shouldn't it be called 'pingpong_recv` if it requires an xcall?

Reply:
no this is to mark this as not using the `recv` modifier